packages/commons/src/CachedContentDownloader.cs

```diff
@@ -2,7 +2,7 @@
 using System.Collections.Concurrent;
 using System.Net;
 
-namespace Deque.AxeCore.Selenium
+namespace Deque.AxeCore.Commons
```

Comment:
I just noticed while looking at this class's implementation that it contains an infinite loop in the constructor (L24). I verified that just attempting to construct it from a unit test hits a StackOverflowException reliably.

Between that and the warning you get when trying to use it that the `WebClient` class it's based on is deprecated, I think that's probably a pretty good indicator that it's not necessary for us to maintain this code going forward. I think it would probably be better to actually just remove this and the associated `ContentDownloader` stuff entirely.

Reply:
(this is a good reminder of the importance of adding unit tests even for these relatively straightforward classes. I'm fine with either including those in this PR or filing a new tracking issue to make sure we don't forget to do it in a separate PR)

Reply:
Tracking with #48 and will most likely opt for a second PR addressing this